### ISSUE TITLE
Admin approval

### ISSUE THREAD

**Issue by nathankota** [agrc/roadkill-mobile#81]
Add a way for Admin to approve new Contractor and Agency users in the database. Both roles can submit data _before_ they are admin approved.

**Comment by stdavis** [comment]
I think that this is a duplicate of #82, isn't it?

**Comment by nathankota** [comment]
Yes, I think they were meant to be different at some point, but have merged into the same thing. I wanted to capture that the contractor and agency roles can submit _before_ they are approved. Is that already implemented, or should I change this issue to describe that?

**Comment by stdavis** [comment]
No, I haven't done anything with data submissions. So changing this issue or opening a new one to reflect that would be great.
